src/vma/config_scanner.c

```diff
@@ -1,5 +1,5 @@
 
-#line 3 "config_scanner.c"
+//#line 3 "config_scanner.c"
```

Comment:
@OphirMunk I would like to remove these lines instead of commenting out to make code clearer. Agree?